File: webview2/app/src/main/java/com/gsolano/webview/SplashActivity.java
package com.gsolano.webview;

import android.content.Intent;
import android.os.Bundle;
import android.view.WindowManager;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.airbnb.lottie.LottieAnimationView;

public class SplashActivity extends AppCompatActivity {


    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        //getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,WindowManager.LayoutParams.FLAG_FULLSCREEN);

        new android.os.Handler().postDelayed(new Runnable() {
                    public void run() {
            Intent mainActivity = new Intent(getApplicationContext(), MainActivity.class);
            mainActivity.addFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION);
            //animationView.cancelAnimation();
            //startActivity(mainActivity);

            startActivityForResult(mainActivity, 0);
            overridePendingTransition(0,0);
        }
        },500);
    }

    @Override
    protected void onStop() {
        super.onStop();
        finish();
    }
}

File: KotlinEjemplos/BaseballUI/app/src/main/java/com/gsolano/baseballui/SplashActivity.kt
package com.gsolano.baseballui

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.view.animation.AnimationUtils
import kotlinx.android.synthetic.main.activity_splash.*

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        var animation= AnimationUtils.loadAnimation(this,R.anim.animation)
        splashActivityIvLogo.startAnimation(animation)
        splashActivitytxtBienvenida.startAnimation(animation)

        var handler = Handler().postDelayed({
            startActivity(Intent(this,IntroActivity::class.java))
        },3000)
    }
}
